solara/components/details.py

```diff
@@ -4,6 +4,41 @@
 
 @solara.component
 def Details(summary="Summary", children=[], expand=False):
+    """Creates an expandable/collapsible section with a summary and additional children content
+
+    ```solara
+    import solara
+
+
+    show_message = solara.reactive(True)
+    disable = solara.reactive(False)
+
+
+    @solara.component
+    def Page():
+       summary_text = "Click to expand for more details"
+        additional_content = [
+            "Additional detail 1",
+            "Additional detail 2",
+            "Additional detail 3"
+        ]
+
+        solara.Details(
+            summary=summary_text,
+            children=additional_content,
+            expand=False
+        )
+
+    ```
+
+
+    ## Arguments:
+
+    *summary: String showing the summary text for the expandable section: Defaults "Summary"
+    *children: List showing the children content of the expandable section: Defaults to an Empty list
+    *expand: Boolean showing if the section is expanded or collapsed: Defaults to False
```

Comment:
Sorry for getting so late at this. But I think it would be good to try to render the docs, it's documented at https://solara.dev/documentation/advanced/development/contribute

I think this is not valid markdown, I think there should be at least a space after the *